app/controllers/articles_controller.rb

```diff
@@ -12,26 +12,25 @@ def feed
     if params[:username]
       if @user = User.find_by_username(params[:username])
         @articles = Article.where(published: true, user_id: @user.id).
-          includes(:user).
-          select(:published_at, :slug, :processed_html, :user_id, :organization_id, :title).
+          select(:published_at, :processed_html, :user_id, :organization_id, :title, :path).
```

Comment:
Not sure if intended but this no longer has `:slug`, while the `elsif` does.

Reply:
Oh yeah, not intended.